simpa_tests/automatic_tests/test_bandpass_filter.py

```diff
@@ -15,51 +15,80 @@
 
 class TestBandpassFilter(unittest.TestCase):
 
```

Comment:
Would it not make sense to add a test, where we take a randomly generated broadband signal and filter it with the implemented filters and then check if the frequency spectrum of the resulting signal matches the filter thresholds chosen (within the margins of the respective method)?

Reply:
Adding an additional test is always a good idea 😄 Especially because the current test case is very artificial. 
How would you determine the tolerance threshold for accepting the test as being successful? As the input is random, we probably can't be too strict but at the same time want to see the test failing if the filtering is not performed optimally.

As this test case is independent of the implementation of the current butterworth filter, I'd suggest to create a new issue for writing this test.

Reply:
I agree with adding a new test but I think the test can be implemented within this issue since new features should be implemented with a corresponding test anyway. :)